index.html

```diff
@@ -1353,6 +1377,15 @@ <h2 id="AudioContext">
             The error that occurred while decoding.
           </dd>
         </dl>
+        <dl title="dictionary AudioContextInit" class="idl">
+          <dt>
+            AudioContextPlaybackCategory bufferingHint
+          </dt>
+          <dd>
+            Provide the hint for how the user agent should trade off between
+            latency and power consumption.
```

Comment:
s/latency/audio output latency/ I think, there are a lot of other latencies everywhere, best to be specific.